STYLEGUIDE_CONVENTION.md

```diff
@@ -0,0 +1,199 @@
+# Code style guide
+
+## Table of Contents
+
+1. [React](#react)
+2. [TypeScript](#typescript)
+3. [CSS](#css)
+
+## React
+
+### JSX
+
+Prefer normal functions (non arrow functions or classes):
```

Comment:
can you explain why we should not use arrow functions? i personally prefer to use them and see that at least in all other repos i've worked before, it's the convention. in anyway, please rewrite line 13 because it's a bit confusing.